staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -151,7 +152,14 @@ func (a *AdmissionOptions) ApplyTo(
 	if err != nil {
 		return err
 	}
-	genericInitializer := initializer.New(clientset, dynamicClient, informers, c.Authorization.Authorizer, features, c.DrainedNotify())
+	var schemaResolver resolver.SchemaResolver
+	if c.OpenAPIConfig != nil && c.OpenAPIConfig.GetDefinitions != nil {
+		schemaResolver = resolver.NewDefinitionsSchemaResolver(k8sscheme.Scheme, c.OpenAPIConfig.GetDefinitions)
+	} else {
+		// OpenAPI config is not initialized, fallback to the discovery resolver.
+		schemaResolver = &resolver.ClientDiscoveryResolver{Discovery: clientset.DiscoveryClient}
```

Comment:
is this fallback exercised in tests and do we understand the performance implications of passing this into admission? if not, limiting ourselves to the built-in resolver seems safer